### PR TITLE
Book updates end may

### PR DESCRIPTION
**Pull request by unixpi**

### CODE REVIEW

File: docs/the_nimbus_book/src/.health.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Monitor the health of your node
```
Comment (bauerj) [inline docs/the_nimbus_book/src/.health.md]: Hmm, should this file be here?

File: docs/the_nimbus_book/src/contribute-network.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Contribute without a validator (WIP: UNPUBLISHED)
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/contribute-network.md]: Let's leave this file in a hackmd for the time being until the tips herein become actionable through the project

File: docs/the_nimbus_book/src/eth1.md [hunk 3]
```diff
@@ -1,17 +1,23 @@
 # Run an eth1 node
 
+> ⚠️ Warning: make sure you've copied the endpoint that starts with either `ws` or `wss` (websocket), and not `http` or `https`. Nimbus does not currently support `http` endpoints.
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/eth1.md:3]: @zah I thought I saw something on https support being added, where is this on the release schedule?

File: docs/the_nimbus_book/src/eth1.md [hunk 4]
```diff
@@ -1,17 +1,23 @@
...
-> As it stands it may be a little complicated to run a full Geth node on a Pi (and light mode doesn't give you the deposit data you need).
->
->In the medium term (3-6 months), we expect someone (perhaps us) will build a thin layer on top of plain Eth1 header-syncing light clients to address this issue. Specifically, what's missing is a gossip network broadcasting deposit proofs (i.e. deposits and corresponding Merkle proofs rooted in Eth1 headers). When that happens, you should be able to swap out Infura.
->
-> However, if you have a > 500GB SSD, and your hardware can handle it, we strongly recommend running your own eth1 client. This will help ensure the network stays as decentralised as possible.
+> **Note:** If you have a > 500GB SSD, and your [hardware](./hardware.md) can handle it, we strongly recommend running your own eth1 client. This will help ensure the network stays as decentralised as possible. If you can't however, the next best option is to set up an [infura endpoint](./infura-guide.md).
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/eth1.md]: let's call them 3rd-party provider generally - there are other options and web services out there than infura that also provide the API - similar to the guide about eth1 clients, we can document more of them generally.

File: docs/the_nimbus_book/src/eth1.md [hunk 5]
```diff
@@ -1,17 +1,23 @@
...
+
+## Nimbus
+In parallel to `nimbus-eth2` we are working hard on our [our eth1 client](https://github.com/status-im/nimbus-eth1). While this is  very much a project in development (i.e. not yet ready for public consumption), you are welcome to experiment with it.
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/eth1.md]:
```suggestion
In parallel to `nimbus-eth2` we are working hard on our [our execution client](https://github.com/status-im/nimbus-eth1). While this is a project in development (i.e. not yet ready for public consumption), we welcome you to experiment with with.
```

File: docs/the_nimbus_book/src/privacy.md [hunk 6]
```diff
@@ -0,0 +1,26 @@
+# Increase privacy
+
+[Do not take your validator privacy lightly](https://our.status.im/validator-privacy-call-to-action/).
+
+## Use a VPN
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/privacy.md]: This is a double-edged sword - you're transferring trust from your ISP to your VPN provider which may be no better, at a latency cost. The jury is still out there on whether this is a good idea, so it's too early to recommend.

File: docs/the_nimbus_book/src/profits.md [hunk 7]
```diff
@@ -0,0 +1,69 @@
...
+
+You can also use the `nimbus-eth2` [API](./api.md). For example, to check if your validator has a next Proposal slot assigned, run:
+
+```
+curl -d '{"jsonrpc":"2.0","method":"get_v1_validator_duties_proposer","params":[${EPOCH_NUMBER_OF_INTEREST}],"id":1}' -H 'Content-Type: application/json' localhost:9190 -s | jq ".result[]" | grep ${PATTERN_WHICH_MATCHES_VALIDATOR_PUBLIC_KEYS}
+```
+
+### Bash script
+
+Quick and dirty bash monitoring/Nimbus-restarting loop:
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/profits.md]:
```suggestion
Script to monitor the logs and restart after when the next action is minutes away:
```

It's a script like any other - though it probably needs some touching up before we recommend it in documentation (@stefantalpalaru) - ie shell shebang etc.

Ideally we would write it for people running nimbus through `systemd` - this is the recommended way to run nimbus in any case (launching through shell script in terminal is just for testing), tying this guide together with the others

File: docs/the_nimbus_book/src/profits.md [hunk 8]
```diff
@@ -0,0 +1,69 @@
...
+Launching the beacon node with the `--subscribe-all-subnets` option increases bandwidth and cpu usage, but helps the network and makes the block production algorithm perform slightly better.
+
+To elaborate a little, without this option enabled Nimbus only listens to a subset of the attestation traffic - in particular, Nimbus doesn't listen to all unaggregated traffic but instead relies on peers to aggregate attestations on the subnets it doesn't subscribe to. 
+
+With this option enabled, Nimbus listens to all unaggregated channels (subscribes to all subnets). Practically speaking, this means that when producing a block, Nimbus can "top up" the aggregates that other peers have made with it's own unaggregated attestations. This can lead to better packing in some cases, which can lead to slightly greater rewards.
+
+
+
+
+## Keep track of your attestation effectiveness
```
Comment (arnetheduck) [inline docs/the_nimbus_book/src/profits.md]: This section really overlaps with general health monitoring, and would probably make more sense in that section - the two are tightly intertwined and should link to each other.

File: docs/the_nimbus_book/src/eth1.md (hunk 3 above)
Comment (unixpi) [reply docs/the_nimbus_book/src/eth1.md:3]: I think Eugene and Jankgo are working in parallel on features that will enable it? We had someone complain that this was unclear and resulted in a lot of time wasted, so best to make it explicit until then.

File: docs/the_nimbus_book/src/profits.md (hunk 7 above)
Comment (unixpi) [reply docs/the_nimbus_book/src/profits.md]: Agree. Removed it for now.

Comment (tersec) [reply docs/the_nimbus_book/src/profits.md]: Fine with either including or removing it, but to clarify, it works fine with systemd. I use it with systemd.

**Pull request merged**